silk-workbench/silk-workbench-workspace/app/controllers/projectApi/ProjectTaskApi.scala

```diff
@@ -28,6 +30,32 @@ import scala.util.Try
 @Tag(name = "Project tasks", description = "Access to all tasks in a project.")
 class ProjectTaskApi @Inject()() extends InjectedController with UserContextActions with ControllerUtilsTrait {
 
+    //validate the task id field by ensuring it's unique and corresponds to the right format
```

Comment:
`@Operation` annotation missing. It should at least contain that an already existing project ID will lead to a 409.